Add ongoing duration metric to Airflow integration (#16847)
DataDog/integrations-core: airflow/tests/README.md

docker exec -it dd_airflow_py39 agent dogstatsd-stats
```

# Airflow REST API metrics

> ## Triggering Dag Task

Reviewer: ```suggestion
## Triggering the DAG Task
```

```bash
ddev env start airflow py3.11-2.6
```

> Go to localhost:8080/login and enter admin credentials

Reviewer: ```suggestion
Go to localhost:8080/login and enter admin credentials.
```

> ### Trigger dag run via UI manually

Reviewer: ```suggestion
### Manually trigger the DAG run via the UI
```

> Triggering the tag should cause the DagRun to start and move it to a running state.

Reviewer: ```suggestion
Trigger the tag to start DagRun and move it to a running state.
```
